python/mxnet/gluon/rnn/rnn_layer.py

```diff
@@ -560,14 +494,10 @@ class GRU(_RNNLayer):
     """
     def __init__(self, hidden_size, num_layers=1, layout='TNC',
                  dropout=0, bidirectional=False, input_size=0,
-                 i2h_weight_initializer=None, h2h_weight_initializer=None,
-                 i2h_bias_initializer='zeros', h2h_bias_initializer='zeros',
```

Comment:
we will need an initializer for the fused parameter and use it as default. With this default initializer for RNN layers, the bias terms should be initialized as 0s.